src/library/scala/math/BigDecimal.scala

```diff
@@ -539,19 +539,19 @@ extends ScalaNumber with ScalaNumericConversions with Serializable {
 
   /** Less-than-or-equals comparison of BigDecimals
    */
-  def <= (that: BigDecimal): Boolean = compare(that) <= 0
+  override def <= (that: BigDecimal): Boolean = compare(that) <= 0
```

Comment:
These all re-implement exactly the same operations as in `Ordered`.  Is there any compelling reason to keep them?